boards/arm/mec172xevb_assy6906/mec172xevb_assy6906.dts

```diff
@@ -130,25 +145,92 @@
 	};
 };
 
+&i2c00_scl_gpio004 {
+	drive-open-drain;
+	output-high;
+};
+
+&i2c00_sda_gpio003 {
+	drive-open-drain;
+	output-high;
+};
+
 &i2c_smb_1 {
 	status = "okay";
 	label = "I2C1";
 	port_sel = <1>;
+	pinctrl-0 = <&i2c01_scl_gpio131 &i2c01_sda_gpio130>;
+	pinctrl-names = "default";
+};
+
+&i2c01_scl_gpio131 {
+	drive-open-drain;
+	output-high;
+};
+
+&i2c01_sda_gpio130 {
+	drive-open-drain;
+	output-high;
 };
 
 &i2c_smb_2 {
 	status = "okay";
 	label = "I2C7";
 	port_sel = <7>;
+	pinctrl-0 = <&i2c07_scl_gpio013 &i2c07_sda_gpio012>;
+	pinctrl-names = "default";
+};
+
+&i2c07_scl_gpio013 {
+	drive-open-drain;
+	output-high;
+};
+
+&i2c07_sda_gpio012 {
+	drive-open-drain;
+	output-high;
 };
 
 &spi0 {
 	status = "okay";
 	clock-frequency = <4000000>;
 	lines = <4>;
 	chip-select = <0>;
+	port-sel = <0>; /* Shared SPI */
+
+	pinctrl-0 = < &shd_cs0_n_gpio055
+		      &shd_clk_gpio056
+		      &shd_io0_gpio223
+		      &shd_io1_gpio224
+		      &shd_io2_gpio227
+		      &shd_io3_gpio016 >;
+	pinctrl-names = "default";
 };
 
 &kscan0 {
 	status = "okay";
+
```

Comment:
@jvasanth1 since same driver will be used for MEC152x and MEC172x.
Driver should compile cleanly for both mec152x boards, that's not the case

Reply:
@jvasanth1 will issue separately, need to understand why CI is not catching the issue